### DISCUSSION TITLE
Service/MySQL: Open TablePlus

### DISCUSSION THREAD

**Discussion by rtrudel** [beyondcode/herd-community#637] [Ideas]
Have a way to open the db in TablePlus like DBngin has already
Just a shortcut to open TablePlus :)

**Comment by mpociot** [comment]
https://x.com/marcelpociot/status/1777315853068857376?s=46&t=cUkCtO-4PC7LZLkfo1H0xA

🙂

**Reply by rtrudel** [reply]
Excellent!!!
